### ISSUE TITLE
Suggestion: Add a sample to README

### ISSUE THREAD

**Issue by jzohrab** [infews/anchorman#9]
Looks like it could be useful, as part of the elevator pitch a sample would be nice so that I know what I'm getting without installing/running etc.

**Comment by npazo** [comment]
I am pretty sure this project is mostly abandoned at this point, but I am thinking about taking my fork of it and getting it up to date.

There are examples here though: https://github.com/infews/anchorman/tree/master/release_notes

It is pretty much just markdown (or HTML) that makes this more human readable. It's fairly easy to customize what you see to what you want though.

**Comment by jzohrab** [comment]
Right, I just saw a link to this today so assumed it was fresh, I should have checked the commit history.  Last commit was oct 2013.  Thanks @npazo.  I'll still leave this open in case the maintainer gets back here.
